R/zchunk_L241.en_newtech_nonco2.R

```diff
@@ -1,107 +1,286 @@
 #' module_emissions_L241.en_newtech_nonco2
 #'
-#' Briefly describe what this chunk does.
+#' Non-CO2 emission coefficients.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
-#' the generated outputs: \code{L241.nonco2_tech_coeff}, \code{L241.nonghg_max_reduction}, \code{L241.nonghg_steepness}. The corresponding file in the
+#' the generated outputs: \code{L241.nonco2_tech_coeff}, \code{L241.nonco2_max_reduction}, \code{L241.nonco2_steepness}. The corresponding file in the
 #' original data system was \code{L241.en_newtech_nonco2.R} (emissions level2).
-#' @details Describe in detail what this chunk does.
+#' @details Generate input tables of non-CO2 emission coefficients.
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
-module_emissions_L241.en_newtech_nonco2_DISABLED <- function(command, ...) {
+#' @author KD August 2017
+module_emissions_L241.en_newtech_nonco2 <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "common/GCAM_region_names",
              FILE = "emissions/A_regions",
              FILE = "energy/A_regions",
              FILE = "emissions/A41.tech_coeff",
              FILE = "emissions/A51.max_reduction",
              FILE = "emissions/A51.steepness",
-             "L111.nonghg_tgej_R_en_S_F_Yh",
-             "L112.ghg_tgej_R_en_S_F_Yh"))
+             FILE = "temp-data-inject/L111.nonghg_tgej_R_en_S_F_Yh",
+             FILE = "temp-data-inject/L112.ghg_tgej_R_en_S_F_Yh"))
   } else if(command == driver.DECLARE_OUTPUTS) {
     return(c("L241.nonco2_tech_coeff",
-             "L241.nonghg_max_reduction",
-             "L241.nonghg_steepness"))
+             "L241.nonco2_max_reduction",
+             "L241.nonco2_steepness"))
   } else if(command == driver.MAKE) {
 
     all_data <- list(...)[[1]]
 
     # Load required inputs
     GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
-    A_regions <- get_data(all_data, "emissions/A_regions")
-    A_regions <- get_data(all_data, "energy/A_regions")
+    Emiss_A_regions <- get_data(all_data, "emissions/A_regions")
+    En_A_regions <- get_data(all_data, "energy/A_regions")
     A41.tech_coeff <- get_data(all_data, "emissions/A41.tech_coeff")
     A51.max_reduction <- get_data(all_data, "emissions/A51.max_reduction")
     A51.steepness <- get_data(all_data, "emissions/A51.steepness")
-    L111.nonghg_tgej_R_en_S_F_Yh <- get_data(all_data, "L111.nonghg_tgej_R_en_S_F_Yh")
```

Comment:
Because this will become available with #613 should I remove the temp-data-inject call?